public/help/html-help.html

```diff
@@ -4,7 +4,7 @@ <h3>Allowed HTML</h3>
 <p>
   <code>a, abbr, acronym, address, [align], [alt], [axis], b, big, blockquote, br, caption, center, cite, [class], code, 
     col, colgroup, dd, del, dfn, div, dl, dt, em, figcaption, figure, h1, h2, h3, h4, h5, h6, [height], hr, [href], i, img, 
-    ins, kbd, li, [name], ol, p, pre, q, s, samp, small, span, [src], strike, strong, sub, sup, table, tbody, td, 
+    ins, kbd, li, [name], ol, p, pre, q, rp, rt, ruby, s, samp, small, span, [src], strike, strong, sub, sup, table, tbody, td, 
```

Comment:
There's another list at

https://github.com/otwcode/otwarchive/blob/2d92ec1e100d2db8d98c791258b8ea115044975d/app/helpers/application_helper.rb#L67-L74

but I don't think we ever set `show_list = true` anywhere. Can you remove that option and the conditional that uses it?

Reply:
For all I could find, it was disabled in 1d9f656ba594d9f2940f26fac8fd99d67409053d (in 2011).

The option is used in app/views/bookmarks/_bookmark_form.html.erb and app/views/comments/_comment_form.html.erb (with show_list=false). I feel like I would be touching parts of the code that's not particularly related to the issue (ruby annotations), but if that's okay, sure, I'll do that.

Reply:
I think it's fine to make these changes. It's unused code that looks puzzling if not updated to include to new tags.